README.md

```diff
@@ -77,6 +77,18 @@ If a base package name is not provided, it will be used the default `org.openapi
 
 Configuring `additional-model-type-annotations` will add all annotations to the generated model files (extra details can be found in [OpenApi Generator Doc](https://openapi-generator.tech/docs/generators/java/#config-options)).
 
+You can customize the name of generated classes. To do that, you must define the properties you need
+
+```properties
+quarkus.openapi-generator.codegen.spec.petstore_json.api-name-suffix=CustomApiSuffix
```

Comment:
We need prefix for the API as well.

Reply:
I can't because `CodegenConfigurator configurator` inside `OpenApiClientGeneratorWrapper` has only these methods:
- setApiNameSuffix
- setModelNameSuffix
- setModelNamePrefix